source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,197 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can configure Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having to 
```

Comment:
Consider, 

"Configuring Relational Migrator with Confluent Cloud gives your deployment the reliability of Kafka without requiring that you manage your own Kafka cluster."

Reply:
Updated.